awx/main/conf.py

```diff
@@ -232,15 +232,6 @@
     category_slug='jobs',
 )
 
-register(
-    'AWX_PROOT_ENABLED',
-    field_class=fields.BooleanField,
-    label=_('Enable job isolation'),
-    help_text=_('Isolates an Ansible job from protected parts of the system to prevent exposing sensitive information.'),
-    category=_('Jobs'),
-    category_slug='jobs',
-)
-
 register(
     'AWX_PROOT_BASE_PATH',
```

Comment:
Are we deleting `AWX_PROOT_BASE_PATH` too?

Reply:
@jakemcdermott *technically* we still lay these files down in `/tmp` before runner is invoked, and there may be cases where customers want to put them on another volume.  So I suspect these settings - while slightly esoteric - probably should continue to exist for now.

Reply:
Don't suppose we could rename them? :D

Reply:
Does the container code still use SHOW_PATHS to map volumes from the host into the container?

Reply:
Yes:

https://github.com/ansible/awx/blob/devel/awx/main/tasks.py#L911-L914

We could probably get rid of `HIDE_PATHS`, though, because...it's a container.